Activities/FTP/UiPath.FTP/SftpSession.cs

```diff
@@ -212,6 +212,52 @@ private async Task<IEnumerable<FtpObjectInfo>> GetRemoteListingAsync(string remo
             return listing;
         }
 
+        private IEnumerable<string> GetMissingDirectories(string remotePath)
```

Comment:
The two methods are almost similar, can't we avoid duplicated code?

Reply:
If you mean the sync and async versions of the same method, there aren't many options. It would be bad practice to wrap one around the other.